src/screens/Server.js

```diff
@@ -157,23 +241,65 @@ class Server extends React.Component {
   }
 
   /**
-   * Update state of the selected server
+   * Update state of the selected base server
    *
    * @param {Object} e Event of select change
    */
-  handleSelectChange = (e) => {
-    this.setState({ selectedValue: DEFAULT_SERVERS[e.target.value] });
+  handleBaseURLSelectChange = (e) => {
+    if (e.target.value === '') {
+      return this.setState({
+        selectedWsServer: '',
```

Comment:
Shouldn't this clean `selectedServer` instead of `selectedWsServer` state?

Reply:
Yes it should, thanks

Fixed in c41501ed53a74ef44d72fc34e64d15367cd17c67